storm/src/main/storm/mesos/NimbusMesosScheduler.java

```diff
@@ -154,13 +160,34 @@ private void updateLogviewerState(TaskStatus status) {
     }
   }
 
-  private void checkRunningLogviewerState(String logviewerZKPath) {
+  private void ensureZNodeExists(String logviewerZKPath) {
```

Comment:
Since this is more for the Logviewer ZNode, can we rename it from `ensureZNodeExists` to `ensureLogviewerZnodeExists`?